### PR TITLE
dev/translation#90 address translation issue on membership

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
_A brief description of the pull request. Keep technical jargon to a minimum. Hyperlink relevant discussions._

Before
----------------------------------------
_What is the old user-interface or technical-contract (as appropriate)?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

After
----------------------------------------
_What changed? What is new old user-interface or technical-contract?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

Technical Details
----------------------------------------
_If the PR involves technical details/changes/considerations which would not be manifest to a casual developer skimming the above sections, please describe the details here._

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

File: templates/CRM/Contribute/Form/Contribution/ThankYou.tpl [hunk 1]
```diff
@@ -116,13 +116,9 @@
           {if !empty($auto_renew)} {* Auto-renew membership confirmation *}
             {crmRegion name="contribution-thankyou-recur-membership"}
               <br />
-              {if !$installments || $installments > 1}
```
Comment (seamuslee001) [inline templates/CRM/Contribute/Form/Contribution/ThankYou.tpl:119]: only thought here @eileenmcnaughton I wonder is if $installments is always being assigned to the template or not

Comment (eileenmcnaughton) [reply templates/CRM/Contribute/Form/Contribution/ThankYou.tpl:119]: @seamuslee001 yeah - I'm confused - I'm thinking it maybe is not - but note that if it is NOT assigned there is still text rendered - it's just that there is

`   <strong>{ts 1=$frequency_unit}This membership will be renewed automatically every %1.{/ts}</strong>`

vs the per-unit version 

& it seems like the per unit version is generally preferred

Comment (eileenmcnaughton) [reply templates/CRM/Contribute/Form/Contribution/ThankYou.tpl:119]: There is also a problem with 

`  <strong>{ts 1=$frequency_interval 2=$frequency_unit}This membership will be renewed automatically every %1 %2(s).{/ts}</strong>`

that seems to be pushed out - I don't know whether you can use the plural whatsit & have day translate to days in English - that seems like it has been left out of scope over a few iterations

Comment (mlutfy) [reply templates/CRM/Contribute/Form/Contribution/ThankYou.tpl:119]: If I read this right, the `every %1 %2(s)` that is still there only affects when a limit is set on the number of instalments, which presumably is not used much.

Ideally it should use the same pattern as for unlimited instalments, with the if/elseif for day/week/month/year (but that's outside the scope of this PR).

Comment (eileenmcnaughton) [reply templates/CRM/Contribute/Form/Contribution/ThankYou.tpl:119]: @mlutfy I pushed in an addition fix for that - if it looks good I suggest we merge this to the rc but only back port the original patch to 5.78

Comment (mlutfy) [reply templates/CRM/Contribute/Form/Contribution/ThankYou.tpl:119]: @eileenmcnaughton  Looks good to me, thanks! I added merge-ready based on the feedback from Samuele on Gitlab (he had also worked on the initial fixes around this).

**Comment by mlutfy** [comment]
I tested a few scenarios and seems all OK. It was on Standalone, with PHP notices enabled, latest Smarty, etc.

Before:

image

After:

image

Contribution Page ThankYou with a number of instalments:

image

Contribution Page ThankYou with no limit of instalments:

image

**Pull request merged**